env/kube_test.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2016 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package env
+
+import (
+	"os/user"
+	"testing"
+
+	"k8s.io/client-go/1.5/kubernetes"
+	"k8s.io/client-go/1.5/pkg/api"
+	"k8s.io/client-go/1.5/pkg/api/v1"
+
+	"istio.io/manager/model"
+	"istio.io/manager/test"
+)
+
+func TestCamelKabob(t *testing.T) {
+	if CamelCaseToKabobCase("ExampleNameX") != "example-name-x" {
```

Comment:
OK, I'll start using table-driven tests. Is there a specific benefit to them as opposed to just code?